index.html

```diff
@@ -7,44 +7,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet"/>
-    <link rel="stylesheet" href="./stylesheets/common.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_mobile.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_cards.css">
-    <link rel="stylesheet" type="text/css" media="screen and (max-width: 999px)" href="./stylesheets/M_shower.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_desktop.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_cards.css">
-    <link rel="stylesheet" type="text/css" media="screen and (min-width: 1000px)" href="./stylesheets/D_shower.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <title>Portfolio</title>
 </head>
 <body>
 
     <header>
-        <p><a href="#" class="logo">StAnLEY</a></p>
+        <p><a href="#" >StAnLEY</a></p>
         <nav>
-            <ul class="hamburger">
-                <li class="drop"><a href="#"></a></li>
-                <li class="drop"><a href="#"></a></li>
-                <li class="drop"><a href="#"></a></li>
+            <ul >
+                <li ><a href="#"></a></li>
+                <li ><a href="#"></a></li>
+                <li ><a href="#"></a></li>
```

Comment:
Please add a class to your semantic tags. so that you can have a well-organized and meaningful code.